src/components/Layout/RightPaneLayout/RightPaneLayout.tsx

```diff
@@ -0,0 +1,52 @@
+import { AnimatePresence, motion } from "framer-motion";
+import React, { ReactElement } from "react";
+import { Css, Palette, px } from "../../../Css";
+import { useRightPaneContext } from "./RightPaneContext";
+
+export function RightPaneLayout({ children, paneBgColor }: { children: ReactElement; paneBgColor?: Palette }) {
+  const { isRightPaneOpen, rightPaneContent } = useRightPaneContext();
+
+  // Todo pass as prop?
+  const paneWidth = 450;
+
+  return (
+    <div css={Css.h100.df.$}>
+      <>
+        <motion.div
```

Comment:
@bdow - do we want to wrap the page content in framer here or use css + transitions

Reply:
My personal choice would be to use CSS Transitions whenever possible and avoid framer-motion. I think framer motion has some good use cases, like when needing to do multiple animations together, or attempting to animate from `auto` to `100%`. But typically, I don't see the benefit in framer motion over css transitions.

Reply:
sounds good. I will switch this over - FWIW the `layout="position"` is what I used to resolve the animations being in sync.

Reply:
Removed framer-motion from the page content wrapper and implemented with css transitions